packages/website/src/hooks/api/usePackage.ts

```diff
@@ -1,10 +1,54 @@
-import { getPackage } from '@/helpers/api';
+import { externalLinks } from '@/constants/externalLinks';
 import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import { ApiDocument, ApiPackage } from '@usecannon/api/types';
 
-export function usePackage({ name, tag, preset, chainId }: { name: string; tag: string; preset: string; chainId: number }) {
+const fetchPackageByRef = async ({ queryKey }: { queryKey: string[] }) => {
+  const [, fullPackagePath] = queryKey;
+  try {
+    const response = await axios.get<{ data: ApiPackage }>(`packages/${fullPackagePath}`, {
+      baseURL: externalLinks.API_CANNON,
+    });
+    return response.data.data;
+  } catch (error) {
+    throw new Error('Failed to fetch package', error as Error);
+  }
+};
+
+const fetchPackageByName = async ({ queryKey }: { queryKey: string[] }) => {
+  const [, name] = queryKey;
+  try {
+    const response = await axios.get<{ total: any; data: ApiDocument[] }>(`packages/${name}`, {
+      baseURL: externalLinks.API_CANNON,
+    });
+    return response.data;
```

Comment:
Shouldn't this be `response.data.data` ?
Not sure about the `response` object, but the API on `GET /packages/:packageName` responds with:
```json
{
  status: number,
  total: number,
  data: { ... }
}
```

Reply:
Ah perfect, just wanted to make sure that it is being used correctly on the Components